swagger.js

```diff
@@ -32,6 +32,53 @@ const doc = {
         },
       },
     },
+    definitions: {
+      Order: {
+        $id: '123456',
+        user: 'John Doe',
+        orderItems: [
+          {
+            album: '5f7f1f7a5c3f2b2d9c1b0b9d',
+            quantity: 2,
+          },
+        ],
+        subtotal: 9.99,
+        tax: 0.75,
+        total: 10.74,
+      },
+    },
+
+    OrderList: {
+      $id: '789012',
+      orders: [
+        {
+          user: 'Jane Smith',
+          orderItems: [
+            {
+              album: '5f7f1f7a5c3f2b2d9c1b0b9d',
+              quantity: 1,
+            },
+          ],
+          subtotal: 14.99,
+          tax: 1.25,
+          total: 16.24,
+        },
+        {
+          user: 'Bob Johnson',
+          orderItems: [
+            {
+              album: '5f7f1f7a5c3f2b2d9c1b0b9e',
+              quantity: 3,
+            },
+          ],
+          subtotal: 29.97,
+          tax: 2.5,
+          total: 32.47,
+        },
+      ],
+      count: 2,
+    },
```

Comment:
When I make a call to the `GET /orders` endpoint, I get a response like this:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/7d396a68-73db-4469-bd6d-80149cb529eb)

It is an object with two keys: `orders` and `count`. So we can remove line 52 since there is no top level `id` field.

As well, instead of having to write what an order looks like again, we can potentially just use: `orders: [ { $ref: '#/definitions/Order' } ],`

The only difference might be that the `user` field here is just the ID, vs in the GET endpoint it's the full user :thinking:  in which case maybe we will just have to rewrite the object here after all.